sale_order_deliverable_rate/models/sale_order.py

```diff
@@ -0,0 +1,48 @@
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    deliverable_rate = fields.Float(
+        compute="_compute_deliverable_rate",
+        search="_search_deliverable_rate",
+        digits="Sale order deliverable rate",
+        store=False,
+    )
+
+    no_order_line_need_delivery = fields.Float(
+        compute="_compute_order_line_need_delivery", store=False
```

Comment:
compute method must contain field name. "_compute_no_order_line_need_delivery"
"nb_" would be better to understand than "no_"